package.json

```diff
@@ -33,7 +33,7 @@
     "enzyme-adapter-react-16": "^1.6.0",
     "file-saver": "^1.3.3",
     "fuse.js": "^3.3.0",
-    "graphiql": "^0.13.0",
+    "graphiql": "~0.13.0",
```

Comment:
latest `0.17.0` of this guy seems breaking webpack and babel 🤦 , pinning it to `0.13.x` for now